components/gif_picker/utils/gfycatSdk.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import Gfycat from './gfycat-sdk.umd.bundle.min';
+
+export default new Gfycat({
```

Comment:
This duplicates what's in mattermost-redux? Also, it looks like the `Gfycat` bundle is thus imported twice: do you make an NPM package available instead?